packages/fuel-gauge/src/predicate/predicate-with-script.test.ts

```diff
@@ -65,14 +63,11 @@ describe('Predicate', () => {
 
       await fundPredicate(wallet, predicate, amountToPredicate);
 
-      expect(toNumber(await predicate.getBalance())).toEqual(
-        initialPredicateBalance + amountToPredicate
-      );
+      expect(toNumber(await predicate.getBalance())).toBeGreaterThan(initialPredicateBalance);
```

Comment:
Mostly because of this: https://github.com/FuelLabs/fuels-ts/issues/2081

We have predicates being used by multiple test cases and since the tests are running concurrently, every type of verification that tries to predict a predicate balance will be flaky.

But `toBeGreaterThan` it is less flaky than `toEqual`

Reply:
I reckon we don't need that mate. 

Many of these balance validations were added only as additional validations, and most do not add value to the test context. I've added many of them. 

Not that our tests are running concurrently, these validation become problematic since we have multiple tests using our predicates